src/AmsterdamPHP/Bundle/MeetupBundle/Command/CreateJoindinPlaceholderCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AmsterdamPHP\Bundle\MeetupBundle\Command;
+
+use CL\Slack\Payload\ChatPostMessagePayload;
+use Codeliner\ArrayReader\ArrayReader;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateJoindinPlaceholderCommand
+ */
+class CreateJoindinPlaceholderCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \AmsterdamPHP\Bundle\MeetupBundle\Service\EventService
+     */
+    protected $meetup;
```

Comment:
nothing deeper honestly. I just code protected by default..... no big philosophy on my end. Private and final seem so... final. This is an OSS project and very collaborative, i don't want anything to slow down anyone contributing.